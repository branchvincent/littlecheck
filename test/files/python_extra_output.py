# RUN: /usr/bin/python3 %s

import sys

print("First")
# CHECK: First

print("Second")
