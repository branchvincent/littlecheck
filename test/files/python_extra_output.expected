Failure in python_extra_output.py:

  There were no remaining checks left to match stdout:2:
    Second

  Context:
    First
    Second <= no more checks

  when running command:
    /usr/bin/python3 python_extra_output.py
